cmd/list_test.go

```diff
@@ -0,0 +1,491 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package cmd
+
+import (
+	"net/http"
+	"reanahub/reana-client-go/utils"
+	"testing"
+
+	"github.com/go-gota/gota/series"
+
+	"golang.org/x/exp/slices"
+)
+
+var listServerPath = "/api/workflows"
+
+func TestList(t *testing.T) {
+	successResponse := `{
+		"total": 2,
+		"items": [
+			{
+				"created": "2022-07-28T12:04:37",
+			  	"id": "my_workflow_id",
+			  	"launcher_url": "https://test.test/url",
+			  	"name": "my_workflow.23",
+			  	"progress": {
+					"finished": {
+					  	"job_ids": ["job1", "job2"],
+					  	"total": 2
+					},
+					"total": {
+						"job_ids": [],
+						"total": 2
+					},
+					"run_finished_at": "2022-07-28T12:13:10",
+					"run_started_at": "2022-07-28T12:04:52"
+			  	},
+			  	"size": {
+					"human_readable": "1 KiB",
+					"raw": 1024
+			  	},
+			  	"status": "finished", 
```

Comment:
Cosmetics: extra trailing white space after "status: finished" line 48. You way want to configure the editor to remove trailing whitespace for all our files.

(The same for line 75 after "status: running".)

Reply:
The editor usually does it but he didn't because it's inside a string. I'll remove them